internal/monitor/driver.go

```diff
@@ -186,3 +186,49 @@ var (
 	deleteLoopBackDevice = utils.DeleteLoopBackDevice
 	unMountPath          = utils.Unmount
 )
+
+//PScaleDriver provides a Driver instance for the PowerScale architecture.
+type PScaleDriver struct {
+}
+
+//GetDriverName returns the driver name string
+func (d *PScaleDriver) GetDriverName() string {
+	return "isilon"
+}
+
+//GetDriverMountDir returns the PowerScale private mount directory.
+func (d *PScaleDriver) GetDriverMountDir(volumeHandle, pvName, podUUID string) string {
+	privateMountDir := fmt.Sprintf("/var/lib/kubelet/pods/%s/volumes/kubernetes.io~csi/%s/mount", podUUID, pvName)
```

Comment:
Is this path still valid if a user decides to change that value : https://github.com/dell/csi-powerscale/blob/main/helm/csi-isilon/values.yaml#L51 to something else that `/var/lib/kubelet` ?

Reply:
Thanks for pointing it out @coulof. Addressed it.